candies/cli/parsers/standard.py

```diff
@@ -88,6 +88,11 @@ def prefixes(signature: inspect.Signature) -> str:
     def prefix_of(x: inspect.Parameter) -> str:
         prefix = getattr(x.annotation, 'prefix', '-')
 
+        if prefix == None:
+            raise ValueError(f"Argument '{x.name}' has an invalid prefix "
+                             f"'{prefix}': "
```

Comment:
Could we concatenate `f"'{prefix}': "` and `f"it is None"` and write them on the same line?

Reply:
I wanted the string to be symmetric to other strings in the function: both are 3 lines tall. ))